roles/idm-host-cert/README.md

```diff
@@ -0,0 +1,100 @@
+Set of Roles
+============
+
+The ansible roles found in this directory are associated with creating and configuring idm host certs.
+
+First it logs into idm, creates a session, and then registers the host. Then, it generates a private key and csr. Next, it creates the cert, retrieves the old one, and overwrites it. As a final check, it prints out the certs.
+
+Requirements
+------------
+
+An IDM accout and ansible.
+
+Role Variables
+--------------
+| Variable | Description | Required | Defaults |
+|:--------:|:-----------:|:--------:|:--------:|
+|**idm_fqdn**| is the fully qualified domain name for the host | no | N/A |
+|**idm_user**| the idm user name | yes | N/A |
+|**idm_password**| the idm password | yes | N/A |
+|**csr_country**| country in which the certificate is made | yes | N/A |
+|**csr_state**| state for the certificate | yes | N/A |
+|**csr_location**| city of the certificate | yes | N/A |
+|**csr_org_name**| organization for the certificate | yes | N/A |
+|**csr_org_unit**| specific team for hte certificate | yes | N/A |
+|**csr_email**| email with which to register the certificate | yes | N/A |
+|**host_name**| the domain name of the host (this is different than the fully qualified domain name) | yes | N/A |
```

Comment:
remove the comment in the parenthesis ... and change this to:
`The hostname/fqdn to be used for the certificate request`